decimal_test.go

```diff
@@ -2542,6 +2542,13 @@ func TestDecimal_Equalities(t *testing.T) {
 	if c.LessThanOrEqual(b) {
 		t.Errorf("%q should not be less than or equal %q", a, b)
 	}
+
```

Comment:
Could you add a separate test case for `IsBetween` where you verify the correctness of left and right bound inclusiveness?